models/models.go

```diff
@@ -84,7 +84,7 @@ func (recipe *Recipe) IsCantabularType() bool {
 }
 
 // validateInstance - checks if fields of OutputInstances are not empty for ValidateAddRecipe and ValidateAddInstance
-func (instance *Instance) validateInstance(ctx context.Context, isCantabularFlexibleTable bool) (missingFields []string, invalidFields []string) {
+func (instance *Instance) validateInstance(ctx context.Context, isCantabularFlexibleTable, isCantabularType bool) (missingFields []string, invalidFields []string) {
```

Comment:
I am not a big fan of polluting the  method signature.
I think it is better to pass the Recipe instance into this method.

```suggestion
func (instance *Instance) validateInstance(ctx context.Context, recipe *models.Recipe) (missingFields []string, invalidFields []string) {
```